tree/ntuple/v7/test/CustomStructLinkDef.h

```diff
@@ -39,6 +39,8 @@
 #pragma link C++ class ConstructorTraits + ;
 #pragma link C++ class DestructorTraits + ;
 
+#pragma link C++ class std::set<std::set<char>> +;
```

Comment:
I wonder whether we should also add all other uses of `std::set` here (e.g. `std::set<CustomStruct>`), @pcanal?

Reply:
Yes.  The dictionary is automatically generated for STL collection only if they are part/used-by another class/struct.

Reply:
Perhaps these should also be moved to another LinkDef file, given that they are unrelated to `CustomStruct`.